models/gw_management_area.py

```diff
@@ -0,0 +1,151 @@
+from django.contrib.gis.db import models
+from gwml2.models.universal import GWTerm
+from gwml2.models.contact_information import CIResponsibleParty
+from gwml2.models.flow.gw_water_budget import GWWaterBudget
+from gwml2.models.time import TMPeriod
+from gwml2.models.fluid_body.gw_fluid_body import GWFluidBody
+from gwml2.models.fluid_body.gw_yield import GWYield
+from gwml2.models.hydrogeological_unit.gw_hydrogeo_unit import GWHydrogeoUnit
+from gwml2.models.universal import DocumentCitation
+
+
+class ManagementAreaTypeTerm(GWTerm):
+    """
+    General classification of the management area
+    (e.g. restricted use zone, irrigation area,
+    consumption area, etc.)
+    """
+    pass
+
+
+class SpecialisedZoneAreaTypeTerm(GWTerm):
+    """
+    Additional classification value which further
+    specialises the gwAreaType.
+    """
+    pass
+
+
+class EnvironmentalDomainTypeTerm(GWTerm):
+    """
+    Classification of the environment domain(s) for
+    which, through the establishment of the
+    management area, certain environmental
+    objectives are to be reached.
+    """
+    pass
+
+
+class GWManagementArea(models.Model):
+    """
+    7.6.26 GW_ManagementArea
+    The GW_ManagementArea represents an area of ground identified for management
+    purposes. The area can be delineated by human factors such as policy or regulation
+    concerns, as well as by domain concerns (in this case hydrogeological or hydrological).
+    The spatial boundaries of a management area do not necessarily align exactly with
+    associated hydrogeological feature boundaries. GW_ManagementArea has the potential
+    to provide a pattern for a more generic OGC 'trans-domain' feature management class.
+    GW_ManagementArea is equivalent to
+    InspireAM:ManagementRestrictionOrRegulationZone.
+    """
+
+    gw_area_name = models.TextField(
+        verbose_name="gwAreaName",
+        help_text="Name of the management area."
+    )
+    gw_area_description = models.TextField(
+        verbose_name="gwAreaDescription",
+        help_text="General description of the management area."
+    )
+
+    # TODO:
+    #  Not sure what Feature is
+    # gw_area_feature = models.ManyToManyField(
+    #     Feature,
+    #     verbose_name="gwAreaFeature",
+    #     help_text="Other features that are associated with the "
+    #               "management area (watershed, ecological zones, etc) "
+    #               "that are not hydrogeological units."
+    # )
+
+    gw_area_water_budget = models.ManyToManyField(
+        GWWaterBudget,
```

Comment:
hi @meomancer  there should be `null=False, blank=False,` because the relationship is
`gwAreaWaterBudget: GW_WaterBudget [1..*]`